pkgs/development/compilers/ponyc/fix-libstdcpp-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/libponyc/CMakeLists.txt b/src/libponyc/CMakeLists.txt
+index bf2c385e..11d0d619 100644
+--- a/src/libponyc/CMakeLists.txt
++++ b/src/libponyc/CMakeLists.txt
+@@ -136,7 +136,7 @@ elseif(${CMAKE_HOST_SYSTEM_NAME} MATCHES "DragonFly")
+ else()
+     # add a rule to generate the standalone library if needed
+     add_custom_command(OUTPUT libponyc-standalone.a
+-        COMMAND cp `find /usr/lib/ -name 'libstdc++.a' -print -quit` libstdcpp.a
++        COMMAND cp `${CMAKE_CXX_COMPILER} --print-file-name='libstdc++.a'` libstdcpp.a
```

Comment:
We can use substituteInPlace for this.

Reply:
Like

```sh
    substituteInPlace src/libponyc/CMakeLists.txt \
        --replace "find /usr/lib/ -name 'libstdc++.a' -print -quit" ''\'''${CMAKE_CXX_COMPILER} --print-file-name="libstdc++.a"'
```

Doesn't feel like a big improvement over a patch that gives context and doesn't require escaping, but I also don't mind changing it.

Reply:
> Doesn't feel

It keeps nixpkgs clean from extra files. If all changes would use patches the repo would be gigantic and we would get more issues with github and git than we already have.

> that gives context

If context is required also add a comment but usually it is pretty clear why something is patched.